src/app/pages/ShipyardPage.jsx

```diff
@@ -298,8 +311,10 @@ export default class ShipyardPage extends Page {
           style={{ height: '1.5em' }}
           className={cn({
             highlighted: noTouch && this.state.shipId === s.id,
+            "compare-highlight": this.state.compare[s.id],
```

Comment:
I might be a bit picky here but is it necessary to have `compare-highlight` in double quotes? I'd prefer if we'd stick to a uniform code style.

Reply:
if not quoted, js will interpret the - as math (minus)
I'll remove the -, renaming it to comparehighlight